charts/ccd-data-store-api/Chart.yaml

```diff
@@ -2,7 +2,7 @@ description: Helm chart for the HMCTS CCD Data Store
 name: ccd-data-store-api
 apiVersion: v2
 home: https://github.com/hmcts/ccd-data-store-api
-version: 1.7.20
+version: 1.7.21
```

Comment:
won't need this, as there is no change in the base chart config.

Reply:
I didn't make this change. It looks to be system generated one.

Reply:
done